pwiz_tools/BiblioSpec/src/BuildParser.cpp

```diff
@@ -43,7 +43,7 @@ BuildParser::BuildParser(BlibBuilder& maker,
     fileroot_ = getFileRoot(fullFilename_);
 
     preferEmbeddedSpectra_ = maker.preferEmbeddedSpectra().get_value_or(true);
-
+    pwiz::util::throw_with_trace(runtime_error("wtf"));

```

Comment:
(clutches pearls)

Reply:
Oh yeah, bit of a test statement there. Probably won't help the blib tests pass. ;)